src/module/os.c

```diff
@@ -92,3 +86,25 @@ void processCwd(WrenVM* vm)
 
   wrenSetSlotString(vm, 0, buffer);
 }
+
+void processExePath(WrenVM* vm)
+{
+  wrenEnsureSlots(vm, 1);
+
+  char buffer[WREN_PATH_MAX * 2 + 1];
```

Comment:
We should really be using `WREN_PATH_MAX` everywhere... does it need to be increased or is there some reason we need much more storage here - in which case I'd suggest we may need another constant vs hard-coding this behavior in multiple places.